src/app/pages/questions/components/question/question.component.ts

```diff
@@ -28,11 +30,26 @@ export class QuestionComponent implements OnChanges {
   value: any
   currentlyShown = false
   previouslyShown = false
+  isLoading = true
+  isScrollable = false
+  margin = 16
+
+  NON_SCROLLABLE_SET: Set<QuestionType> = new Set([
+    QuestionType.timed,
+    QuestionType.audio,
+    QuestionType.info
```

Comment:
Why is info type not scrollable?

Reply:
This is for the outer container. Only the info screen's inner box is scrollable, while for radio buttons, their whole container is scrollable if there are many options, etc.